src/v2/view-builder/views/EnrollProfileView.js

```diff
@@ -5,15 +5,17 @@ import { FORMS as RemediationForms } from '../../ion/RemediationConstants';
 const Body = BaseForm.extend({
   title() {
     if (this.options.appState.getCurrentViewState().href.endsWith('idp/idx/enroll/update')) {
-      return loc('oie.primaryauth.submit', 'login');
+      const label = this.options.appState.attributes.uiDisplay.label;
+      return loc('oie.registration.form.customize.label', 'login', [label]);
```

Comment:
Might be best practice to do null checks on `label` and `buttonLabel`, like ` this.options.appState?.attributes?.uiDisplay?.label ` and make sure it's not `undefined`.

Reply:
what if `uiDisplay` doesn't exists ? we have still have to return some default title right ?

Reply:
Correct, we can define and use default labels from a properties file

Reply:
Thanks for the suggestion! will do that!

Reply:
value check added.